src/picologging/formatter.cxx

```diff
@@ -266,9 +265,10 @@ PyObject* Formatter_repr(Formatter *self)
 }
 
 PyObject* Formatter_dealloc(Formatter *self) {
-    Py_XDECREF(self->style);
     Py_XDECREF(self->fmt);
     Py_XDECREF(self->dateFmt);
+    Py_XDECREF(self->style);
```

Comment:
I assume the move of self->style is just a style thing (<--teehee) or accidental move.

Reply:
yeah i wanted to do it in the right order to make it easier to read.